### DISCUSSION TITLE
多配置文件加载策略问题

### DISCUSSION THREAD

**Discussion by frankli0324** [v2fly/v2ray-core#1339] [Ideas]
起初后加载的配置会覆盖前者，之后对于inbounds和outbounds数组“如果后加载的配置中数组元素个数为1”则附加到inbounds的后面
我并没get到为什么不是“能附加就附加”而是以覆盖为主，从使用的角度来讲个人认为应该“能附加就附加”

please, inspire me.

**Comment by Ch1cC** [comment]
2022了还是没有采取/(ㄒoㄒ)/~~
![可能运行](https://user-images.githubusercontent.com/48575759/191858152-20ca44bc-b6de-4357-a274-398075b33efa.png)
